<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>AZ::SceneAPI::UI::OverlayWidgetLayer</class>
  <widget class="QWidget" name="AZ::SceneAPI::UI::OverlayWidgetLayer">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>75</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
        <widget class="QFrame" name="m_center">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="m_centerLayout">
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="bottomMargin">
              <number>0</number>
            </property>
          </layout>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QFrame" name="m_controls">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>30</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="m_controlsLayout">
            <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
              <number>2</number>
            </property>
            <property name="topMargin">
              <number>2</number>
            </property>
            <property name="rightMargin">
              <number>2</number>
            </property>
            <property name="bottomMargin">
              <number>2</number>
            </property>
            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>